### PR TITLE
Move scene dataset creation to zarr

### PR DESCRIPTION
**Pull request by lucabergamini**
Allow to get a scene dataset from `ChunkedDataset` only. This reduces dependencies on `EgoDataset`.
Add also a test for the new function

### CODE REVIEW

File: l5kit/l5kit/dataset/ego.py [hunk 1]
```diff
@@ -146,25 +145,7 @@ def get_scene_dataset(self, scene_index: int) -> "EgoDataset":
 
         """
         # copy everything to avoid references (scene is already detached from zarr if get_combined_scene was called)
```
Comment (yawei-ye) [inline l5kit/l5kit/dataset/ego.py:147]: The comment above is not clear anymore

File: l5kit/l5kit/data/zarr_dataset.py [hunk 2]
```diff
@@ -198,3 +199,36 @@ def __str__(self) -> str:
         table.float_format = ".2"
         table.add_row(values)
         return str(table)
+
+    def get_scene_dataset(self, scene_index: int) -> "ChunkedDataset":
+        """
+        Get a new ChunkedDataset of a single scene. This dataset lives in memory (as np.ndarray)
```
Comment (yawei-ye) [inline l5kit/l5kit/data/zarr_dataset.py:205]: can we use follow SWE excellency to use same doc style as in jadoo? 
https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html

**Review by yawei-ye — COMMENTED** [review]
Minor, otherwise LGTM

File: l5kit/l5kit/data/zarr_dataset.py (hunk 2 above)
Comment (lucabergamini) [reply l5kit/l5kit/data/zarr_dataset.py:205]: ah yeah I forgot to change from google to sphinx

**Comment by lucabergamini** [comment]
@yawei-ye PTAL

**Review by yawei-ye — APPROVED** [review]
LGTM

**Pull request merged**